src/usr/local/www/pkg_mgr.php

```diff
@@ -130,12 +128,12 @@
 //		display_top_tabs($tab_array);
 }
 
-if (!$pkg_info || !is_array($pkg_info)):
-?>
+if (!$pkg_info || !is_array($pkg_info)):?>
```

Comment:
What would you like that line to look like then? Drop the ?> to the following line?